### ISSUE TITLE
build-x86-images.sh fails

### ISSUE THREAD

**Issue by imv** [voidlinux/void-mklive#74]
I try to build the image in a chrooted Void installation hosted on Arch Linux.
Running `make` and `./build-x86-images.sh -a x86_64-musl -b xfce` fails with an error:

```
[6/[6/9] Generating GRUB support for EFI systems...
ERROR: Failed to move /root/void-mklive/tmp.ChGGRB42fj/void-host as rootfs (Invalid argument)
ERROR: Failed to generate EFI loader
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/dev': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/proc': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/sys': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/dev': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/proc': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/sys': Device or resource busy
9] Generating GRUB support for EFI systems...
ERROR: Failed to move /root/void-mklive/tmp.ChGGRB42fj/void-host as rootfs (Invalid argument)
ERROR: Failed to generate EFI loader
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/dev': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/proc': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/sys': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/dev': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/proc': Device or resource busy
rm: cannot remove '/root/void-mklive/tmp.ChGGRB42fj/void-host/sys': Device or resource busy
```

**Comment by ebfe** [comment]
voidlinux/xbps#152

**Comment by the-maldridge** [comment]
This issue was resolved in a patch that is scheduled to be released on 0.52.  While I normally am opposed to closing issues that reference as yet unreleased code, 0.52 has taken long enough to drop that this could be a very long standing issue.  If you require this functionality, build xbps from master.
